README.md

```diff
@@ -70,42 +70,75 @@ by default for all smartphones & tablets. Heck, it even works on the PlayStation
 
 Vanilla is built to be simple, and its installation is no exception.
 
-* Upload Vanilla's [pre-built version](https://open.vanillaforums.com/addon/vanilla-core) to your server.
-* Confirm the cache, conf, and uploads folders are writable by PHP.
-* Navigate to that folder in your web browser.
-* Follow the instructions on screen.
+1. Upload Vanilla's [pre-built version](https://open.vanillaforums.com/addon/vanilla-core) to your server.
+1. Using nginx? [See our nginx guide](https://docs.vanillaforums.com/developer/backend/server-nginx/).
+1. Confirm the `cache`, `conf`, and `uploads` folders are writable by PHP.
+1. Navigate to the folder where you uploaded Vanilla in your web browser.
+1. Follow the instructions on screen.
+
+If you run into a problem, see [Getting Help](#getting-help) below.
 
 ## Upgrading
 
-Follow these steps to upgrade Vanilla when a new stable release is announced.
+Follow these steps to upgrade Vanilla when a new stable release is announced. These instructions assume you're using SFTP to manually copy files to a server.
+
+Please consider using [maintenance mode](#using-maintenance-mode) before running database updates if your database is very large (millions of users or comments).
+
+1. Backup your database, `.htaccess` and `conf/config.php` file somewhere safe.
+1. Upload the new release's files so they overwrite the old ones.
+1. Delete all files in `/cache`.
```

Comment:
I don't if we have a better solution to this problem, but if deleting all of the files in `/cache` includes deleting an `.htaccess` file currently. If you're on git, it comes up as a diff, and could prevent people from doing a `git pull` in the future without additional interventions.

Worse is that people running on apache with `.htaccess` are then no-longer denying access to the cache folder.

Reply:
I added a note at the top that these instructions are for copy/pasting files, not a git-based setup, fwiw. There's like 8 other things wrong with these instructions if you're using git, nevermind that you shouldn't be using git directly in a web directory.

Reply:
That's true, but this still could open servers up to allow people access the cache directory.

Either that `.htaccess` is useless, and we should get rid of it, or it's important and they shouldn't delete it.